sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaConverter.scala

```diff
@@ -586,11 +586,10 @@ private[sql] object ParquetSchemaConverter {
 
   def checkFieldName(name: String): Unit = {
     // ,;{}()\n\t= and space are special characters in Parquet schema
-    checkConversionRequirement(
-      !name.matches(".*[ ,;{}()\n\t=].*"),
-      s"""Attribute name "$name" contains invalid character(s) among " ,;{}()\\n\\t=".
```

Comment:
`Attribute` -> `Column` ?

Reply:
Not under stand what you mean ==

Reply:
Why you change the message?

Reply:
> Why you change the message?

unify such schema column name error and make same error message

Reply:
So this change makes break change? Maybe we need update migrate docs.